CONTRIBUTING.md

```diff
@@ -0,0 +1,95 @@
+* **If you're reporting it crashed**, include a crash report with a stack trace.
+
+### Suggesting Enhancements
+
+This section guides you through submitting an enhancement suggestion, including completely new features and minor improvements to existing functionality. Following these guidelines helps maintainers and the community understand your suggestion :pencil: and find related suggestions :mag_right:.
+
+#### How Do I Submit A (Good) Enhancement Suggestion?
+
+Enhancement suggestions are tracked as [GitHub issues](https://guides.github.com/features/issues/). Create an enhancement on the repository and provide the following information by filling in [the template](https://github.com/systelab/systelab-components/blob/master/.github/ISSUE_TEMPLATE/feature_request.md).
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Include screenshots** which help you demonstrate the steps or point out the part of library which the suggestion is related to.
+
+### Pull Requests
+
+The process described here has several goals:
+
+- Maintain library's quality
+- Fix problems that are important to users
+- Engage the community in working toward the best possible library
+- Enable a sustainable system for library's maintainers to review contributions
+
+Please follow these steps to have your contribution considered by the maintainers:
+
+1. This repository is aligned with the feature-branch strategy.
+2. Follow all instructions in [the template](PULL_REQUEST_TEMPLATE.md)
+3. After you submit your pull request, verify that all [status checks](https://help.github.com/articles/about-status-checks/) are passing <details><summary>What if the status checks are failing?</summary>If a status check is failing, and you believe that the failure is unrelated to your change, please leave a comment on the pull request explaining why you believe the failure is unrelated. A maintainer will re-run the status check for you. If we conclude that the failure was a false positive, then we will open an issue to track that problem with our status check suite.</details>
+4. While the prerequisites above must be satisfied prior to having your pull request reviewed, the reviewer(s) may ask you to complete additional design work, tests, or other changes before your pull request can be ultimately accepted.
+5. Once it's merged, the author must transition it to the status "Ready to test" in the [project](https://github.com/systelab/systelab-components/projects)
```

Comment:
I have a question about this. Would it be possible to test the component before merging the pull-request? In that case, how should we address this, because the component will not be able to be made public until it has been merged and then updated in the npm repository, right?
Maybe it can be tested in the showcase, can that be enough? If so, I would suggest doing this step (changing to ready to test) before merging the change to avoid having untested components and changes in the master branch (and potentially in the npm package).

Reply:
@josepvilabadillo We do test it with the HemoHub project before publishing. Steps:
1) npm run build-lib
2) The library is generated at "dist\systelab-components
3) Go to your project > node-modules > systelab-components and replace the content. Please, ensure that the package.json matches the version that you need to test. We usually do it with the last published version.
4) Run your project with the new changes

I will also include this in the guide.